Robus/src/reception.c

```diff
@@ -65,8 +65,8 @@ void Recep_GetHeader(volatile uint8_t *data)
     // Check if we have all we need.
     switch (data_count)
     {
-        case 1: //reset CRC computation
-#ifdef SNIFFER_H
+        case 1:  //reset CRC computation
+#ifdef SNIFFER_H //when we catch the first byte we timestamp the msg
```

Comment:
You should put those comment updates on the related commit to make people think that you are a perfect developer. ;)